docs/api-reference/geo-layers/imagery-layer.md

```diff
@@ -0,0 +1,183 @@
+import {ImageryLayerDemo} from 'website-components/doc-demos/geo-layers';
+
+<TileLayerDemo />
```

Comment:
Remove the wrong demo and mark as TODO

Reply:
Removed the example, not sure how to mark as TODO in markdown